pdffigures.cpp

```diff
@@ -219,6 +228,11 @@ int main(int argc, char **argv) {
       graphics1d = std::unique_ptr<PIX>(pixCreateTemplate(fullRender1d.get()));
     }
 
+    std::unique_ptr<PIX> fullColorRender;
+    if(colorImagePrefix.length() != 0){
```

Comment:
One minor nit, rather then having two separate if statements checking the same thing this can just be moved right above the call to `saveFiguresFullColorImage`